app/controllers/controller.rb

```diff
@@ -0,0 +1,91 @@
+
+###### INDEX ######
+
+get '/' do
+  erb :home
+end
+
+# get '/hello' do
+#   erb :new
+# end
+
+get '/questions/new' do
+  # get form to add new question
+
+  erb :create_new_question
+end
+
+post '/questions' do
+  title = params[:title]
+  body = params[:body]
+  programming_language = params[:programming_language]
+  @question = Question.create!(title: title, body: body, programming_language: programming_language)
```

Comment:
you should never use `create!` in your controllers. the bang methods are for us as developers. if they blow up on us (meaning validations fail), our entire application will break. the bang methods should only be used in your seed or test files.